scp/base_melinder.py

```diff
@@ -156,3 +182,13 @@ def density(self, temp: float) -> float:
         """
 
         return self._f_prop(self.coefficient_density(), temp)
+
+    def freeze_point(self, x: float) -> float:
```

Comment:
And here is where we implement the abstract method and this is called the API/CLI method.